functions/go/fix/v1alpha2/types.go

```diff
@@ -0,0 +1,281 @@
+package v1alpha2
```

Comment:
What's the plan for keeping this up-to-date with future changes to v1alpha2 Are you going to regularly update it until we go to `v1`? I don't expect much change, but there are changes common.

Reply:
That's the advantage of making this a function rather than a CLI command. The next patch versions of this function will take care of that. In future if we have 2.0 of kpt, `fix:v0.2` can do that migration.